src/components/SignUp/Landing.tsx

```diff
@@ -61,22 +56,26 @@ const Landing: React.FC<{
         lg="5"
         onClick={setFormCheck}
       >
-        <Image alt="" src={blob} />
-        <div>
-          <p>{text}</p>
-          <div>{input}</div>
-        </div>
+        <Image alt="" src={icon} />
+
+        <div>{text}</div>
+        <div>{input}</div>
       </StyledCard>
     );
   };
 
   const cards = [
     [
       {
-        blob: Green_blob,
+        icon: Individual,
         className: 'green-shadow-hover',
         name: 'individual',
-        text: <span>An Individual</span>,
+        text: (
+          <>
+            <span className="cardHeading">Individual Artist</span>{' '}
```

Comment:
Instead of a class for this, we could actually create a styled component below. So, here we would have:
```
<CardHeading>Off-Stage</CardHeading>
```

And below in the styling, we'd have like:
```
const CardHeading = styled.h3`
  ...your styles here
`;
```

Notice the use of a semantic, block title tag, as well, instead of a span